### DISCUSSION TITLE
An inbuilt Equalizer is required

### DISCUSSION THREAD

**Discussion by godwin-binoy** [fast4x/RiMusic#4744] [Ideas]
An inbuilt Equalizer is required in the application. Most of devices don't have Equaliser option in settings. An inbuilt Equalizer would be beneficial to users who doesn't have Equalizer and who uses third party Equalizer.

Example :
![Screenshot_2024-09-19-12-43-53-56.jpg](https://github.com/user-attachments/assets/ffe1a4f9-e3a7-416f-a6a8-6e003c0bba20)

It's Equalizer in black hole music.
Black hole music is no longer supported.

**Comment by fast4x** [comment]
Hello, it's planned.

**Reply by godwin-binoy** [reply]
That's great 👍

**Comment by ChiefMikeK** [comment]
VLC equalizer is open source, it's probly able to patch into RImusic
- https://github.com/videolan/vlc/blob/master/modules/audio_filter/equalizer.c
